### PR TITLE
Feature/Fetch existing token info & validation - DAO-654

### PR DESCRIPTION
**Pull request by Fabricevladimir**
- fetch token info logic
- validation for token contract address

### CODE REVIEW

**Review by sepehr2github — APPROVED** [review]
Thank you Fabrice. Looks good to me!

**Comment by sepehr2github** [comment]
@Fabricevladimir 
Just a small question. I think we should see contract information when we added the contract. Am I right?

**Review by RakeshUP — APPROVED** [review]

File: packages/web-app/src/containers/setupCommunity/addExistingToken.tsx [hunk 1]
```diff
@@ -37,32 +121,50 @@ const AddExistingToken: React.FC = () => {
...
-          <InfoContainer>
-            <Label label={t('labels.existingTokenSymbol')} />
-            <TextInput disabled value="ANT" />
-          </InfoContainer>
-          <InfoContainer>
-            <Label label={t('labels.existingTokenSupply')} />
-            <TextInput disabled value="43,028,631" />
-          </InfoContainer>
-        </TokenInfoContainer>
+        {name && (
```
Comment (RakeshUP) [inline packages/web-app/src/containers/setupCommunity/addExistingToken.tsx]: @Fabricevladimir even I didn't see the existing token information when I checked the UI

Comment (sepehr2github) [reply packages/web-app/src/containers/setupCommunity/addExistingToken.tsx]: This is because You didn't connect your wallet :)))

**Pull request merged**